web/tls_config_go118_test.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2019 The Prometheus Authors
```

Comment:
Nit, new file, new date.

```suggestion
// Copyright 2023 The Prometheus Authors
```